### PR TITLE
dm: Improve the text help of the Kconfig options

### PR DESCRIPTION
**Pull request by maje-emb**
Introduces changes in the help text of the Kconfig option.

### CODE REVIEW

File: subsys/dm/Kconfig [hunk 1]
```diff
@@ -47,68 +47,68 @@ config DM_HIGH_PRECISION_CALC
 	default n
 	bool "Use high-precision distance estimation"
 	help
-		"Use a more compute-intensive algorithm for the distance estimation.
-		Only works with MCPD ranging mode"
+	  Use a more compute-intensive algorithm for the distance estimation.
+	  Only works with MCPD ranging mode
```
Comment (peknis) [inline subsys/dm/Kconfig:51]:
```suggestion
	  Works only with MCPD ranging mode.
```

File: subsys/dm/rpc/Kconfig [hunk 2]
```diff
@@ -23,9 +23,15 @@ choice DM_MODULE_RPC_ROLE_CHOICE
 
 config DM_MODULE_RPC_HOST
 	bool "Distance Measurement over RPC Host"
+	help
+	  Distance Measurement API use nRF RPC library to provide API for client.
```
Comment (peknis) [inline subsys/dm/rpc/Kconfig:27]:
```suggestion
	  The Distance Measurement API uses the nRF RPC library to provide API for the client.
```

File: subsys/dm/rpc/Kconfig [hunk 3]
```diff
@@ -23,9 +23,15 @@ choice DM_MODULE_RPC_ROLE_CHOICE
...
+	  It operates on a radio core.
 
 config DM_MODULE_RPC_CLIENT
 	bool "Distance Measurement over RPC Client"
+	help
+	  Distance Measurement API use nRF RPC library to access host on remote core.
```
Comment (peknis) [inline subsys/dm/rpc/Kconfig:33]:
```suggestion
	  The Distance Measurement API uses the nRF RPC library to access host on a remote core.
```

File: subsys/dm/rpc/Kconfig [hunk 4]
```diff
@@ -23,9 +23,15 @@ choice DM_MODULE_RPC_ROLE_CHOICE
...
+	  It operates on a application core.
```
Comment (peknis) [inline subsys/dm/rpc/Kconfig:34]:
```suggestion
	  It operates on the application core.
```

File: subsys/dm/Kconfig [hunk 5]
```diff
@@ -9,7 +9,7 @@ config DM_MODULE
 	select NRF_DM
 	select EXPERIMENTAL
 	help
-		Enable DM (Distance Measurement) module.
+	  Enable DM (Distance Measurement) module.
```
Comment (peknis) [inline subsys/dm/Kconfig:12]:
```suggestion
	  Enable the DM (Distance Measurement) module.
```

File: subsys/dm/Kconfig [hunk 6]
```diff
@@ -47,68 +47,68 @@ config DM_HIGH_PRECISION_CALC
...
 	help
-	  "Enable allocation of the next timeslot after the previous timeslot has succeeded."
+	  Enable allocation of the next timeslot after the previous timeslot has succeeded.
 
 config DM_INITIATOR_DELAY_US
 	int "Initiator start delay"
 	default 1000
 	help
-	  "Additional Initiator Start delay time. Used to adjust synchronization."
+	  Additional Initiator Start delay time. Used to adjust synchronization.
```
Comment (peknis) [inline subsys/dm/Kconfig:63]:
```suggestion
	  Additional Initiator Start delay time.
	  Used to adjust synchronization.
```
Not sure whether these need to be on separate lines, but that seems to be the case with other instances.

**Review by peknis — CHANGES_REQUESTED** [review]

File: subsys/dm/Kconfig (hunk 6 above)
Comment (maje-emb) [reply subsys/dm/Kconfig]: Good point.

**Review by peknis — APPROVED** [review]

File: subsys/dm/rpc/Kconfig [hunk 7]
```diff
@@ -23,9 +23,15 @@ choice DM_MODULE_RPC_ROLE_CHOICE
 
 config DM_MODULE_RPC_HOST
 	bool "Distance Measurement over RPC Host"
+	help
+	  The Distance Measurement API uses the nRF RPC library to provide API for the client.
+	  It operates on the radio core.
```
[the remark this answers is not in the document]
Comment (maje-emb) [reply subsys/dm/rpc/Kconfig]: Good point, improved.

File: subsys/dm/rpc/Kconfig [hunk 8]
```diff
@@ -23,9 +23,15 @@ choice DM_MODULE_RPC_ROLE_CHOICE
...
 
 config DM_MODULE_RPC_CLIENT
 	bool "Distance Measurement over RPC Client"
+	help
+	  The Distance Measurement API uses the nRF RPC library to access host on a remote core.
+	  It operates on the application core.
```
[the remark this answers is not in the document]
Comment (maje-emb) [reply subsys/dm/rpc/Kconfig]: ACK

**Review by grochu — APPROVED** [review]
Looks good

**Pull request merged**